charts/sonarqube-dce/Chart.yaml

```diff
@@ -21,27 +21,7 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "Updated SonarQube to 9.7.0"
```

Comment:
you can revert this change as we are not releasing from this branch

Reply:
Right, but this is correct on the branch needed for the release right?

Reply:
yes

Reply:
All reverted!